.github/ISSUE_TEMPLATE.md

```diff
@@ -14,13 +17,23 @@ Version Information:
 
 <!--
 Steps to reproduce the problem:
+
+1. {{add step 1}}
+2. {{add step 2}}
+3. {{add step 3}}
 -->
 
 
 <!--
-Some guidelines:
+Before you report this issue, please consider the following:
```

Comment:
Too verbose, and asking users to "consider" doing way too many things.
How about just adding a fourth point, "For security issues, please contact us at security@getpostman.com, you will receive a response within 24 hours."